src/main/java/com/org/jenkins/custom/jenkins/distribution/service/generators/WarGenerator.java

```diff
@@ -31,10 +27,7 @@ public static void generateWAR(String versionName, String configuration) {
             new Builder(cfg).build();
             LOGGER.info("Cleaning up temporary directory");
             packagerConfigFile.deleteOnExit();
-        } catch (Exception e) {
-            e.printStackTrace();
-        } finally {
-            util.cleanupTempDirectory(tempDirWithPrefix.toFile());
-        }
+            return new File("/tmp/output/target/" + artifactID + "-" + versionName + ".war");
```

Comment:
I think there is a problem here. The path to the war file is hardcoded. You have to get the source of truth for the location of the war file. The `cfg` instance holds that information. Look at the [getOutputWar() method](https://github.com/jenkinsci/custom-war-packager/blob/master/custom-war-packager-lib/src/main/java/io/jenkins/tools/warpackager/lib/config/Config.java#L113) of the `Config` class, I think that that is the source of truth for the war file itself. This way, if `Config.java` changes in the future, you still get the right answer.